app/controllers/users_controller.rb

```diff
@@ -1,13 +1,15 @@
 class UsersController < ApplicationController
-  before_action :authenticate_user!
+  after_action :verify_authorized
 
   def edit
     @user = current_user
+    authorize @user, policy_class: UserPolicy
```

Comment:
Had to specify the policy here since the logged in user could be a `Volunteer` and we still want them to be able to edit themselves.